test/parseLongPath.cpp

```diff
@@ -32,7 +32,6 @@ namespace
 TEST(ParseLongPathTest, invalid)
 {
   ASSERT_THROW(parseLongPath(""), std::runtime_error);
-  ASSERT_THROW(parseLongPath("A"), std::runtime_error);
   ASSERT_THROW(parseLongPath("A/"), std::runtime_error);
   ASSERT_THROW(parseLongPath("AB"), std::runtime_error);
   ASSERT_THROW(parseLongPath("AB/path"), std::runtime_error);
```

Comment:
I think that from the point of view of consistency, the following cases should be treated as equivalent:

- `A`
- `A/`
- `/A`
- `/A/`

Reply:
Yes @veloman-yunkan, There is no point in treating them as invalid cases. Modifying the code to accommodate this and adding unit tests for these cases as well.